.semgrep/adapter/type-bid-assignment.yml

```diff
@@ -0,0 +1,61 @@
+rules:
+  - id: type-bid-assignment-check
+    languages:
+      - go
+    message: >
+      Found incorrect assignment made to `Bid` Recommended to either create copy of `bid` and then assign it as `Bid`  
+
+      ```
+        for _, seatBid := range response.SeatBid {
+          for _, bid := range seatBids.Bid {
```

Comment:
nitpick:- typo here it should be `seatBid.Bid`

Reply:
[e49d669](https://github.com/prebid/prebid-server/pull/2833/commits/e49d669a214a88ded6094d03bf4ea1760b4f63c5) fixes typo